coral/src/app/features/topics/BrowseTopics.tsx

```diff
@@ -0,0 +1,7 @@
+import { TopicList } from "src/app/features/topics/list/TopicList";
+
+function BrowseTopics() {
```

Comment:
I'm thinking about maybe naming it `Topics` or `TopicsView`, because there's already a `TopicsList` in `topics/components`. 

If we build e.g. the search element, the structure than could be: 

```
.
└── features/
    ├── topics/
    │   ├── list/
    │   │   ├── TopicCard.tsx
    │   │   ├── ...etc.
    │   │   └── TopicList.tsx
    │   └── search/
    │       └── TopicSearch.tsx
    └── Topics.tsx
```

In that case, `TopicList` can be seen as a feature that you can use without `TopicSearch` (which you can) and the component on the root brings everything together. 

Just the naming `Topics` is a bit sub-optional because there's already a page `Topics` 🤔  (I think that's why I landed at `BrowserTopics` but I'm not really into that name anyway)

Reply:
Did not think of that... Naming is hard 🤔 . Would it be good approach if the page component would render the heading and list. I have not yet have a clear picture how dumb the page component should be. We can leave it as is, but again, when doing more of these lets try to find the right abstraction and document it.